pxr/imaging/hgiInterop/hgiInterop.h

```diff
@@ -73,10 +67,16 @@ class HgiInterop final
     void TransferToApp(
         Hgi *srcHgi,
         HgiTextureHandle const &srcColor,
+        HgiBlendFactor colorSrcBlendFactor,
```

Comment:
I'm very curious about why GL style fixed function parameters are passed here? In general wouldn't you just want a blit to a target? Fancy compositing really seems like something in the app domain and something an app could take care of in window/widget composition. ie, the viewport widget could just draw a textured quad, as opposed to trying to set up Hydra to do something a bit fancy? Minimally could we at least also have a not-fancy plain old blit? This set up seems like something I wouldn't want to have to deal with in my own apps, for example.

Reply:
Also worth noting that these operators are still part of modern APIs, for example [`VkPipelineColorBlendAttachmentState`](https://docs.vulkan.org/spec/latest/chapters/framebuffer.html#VkPipelineColorBlendAttachmentState) and [`VkPipelineDepthStencilStateCreateInfo`](https://docs.vulkan.org/spec/latest/chapters/fragops.html#VkPipelineDepthStencilStateCreateInfo), since that part of the fragment pipeline, while more customizable, is still pretty much "fixed function".

Reply:
Gotcha. @clach commented on having them in a struct, and if the struct defaults to the existing case, then I'm happy with that.

Reply:
I did add a struct, but I set the defaults as to fully overwrite instead of compositing. I think the default should be the "null" state, and "null composition" should be no composition. Least surprising behaviour.

Reply:
Least surprising is the best :D